client/commonFramework/show-completion.js

```diff
@@ -68,6 +68,70 @@ window.common = (function(global) {
       });
   }
 
+  function submitGithubHandler(e) {
+    e.preventDefault();
+
+    var solution = common.editor.getValue();
```

Comment:
Not sure that mixing `var`, `let` and `const` declarations is a good style. Replace those `var` declarations either with `const` (recommended, if there is no need to reassign the value) or with `let`.